conan-adapter v0.3.1 integration

ChGen opened the pull request:
conan-adapter v0.3.1 integration.

ChGen commented:
@olenagerasimova , please review.

olenagerasimova commented:
@ChGen please, add (as we have for each adapter):
- integration tests for upload and install to/from the adapter
- smoke test for upload and install to/from the adapter
- adapter settings description to wiki

ChGen commented:
@olenagerasimova , thanks, 
I have integration tests in the conan-adapter repo now. Should I move them all to the artipie main repo, or only some part of them?

olenagerasimova commented:
@ChGen no, you should not move or copy any tests from the adapter. Tests in the adapter and artipie differ: adapter tests run adapter slice only, when test in artipie run whole artipie image with corresponding adapter settings file and etc. Please, check any repository test in artipie, for python repo or debian repo as an example.

ChGen commented:
@olenagerasimova , please review. I added Conan integration tests.

olenagerasimova reviewed (commented):
@ChGen thanks, integration test look good to me. What about wiki and smoke test?

ChGen commented:
@olenagerasimova , please review; I added smoke test & basic wiki for conan. Also for integration test, I switched to custom ubuntu image for uniform testing environment and arm64 support.

olenagerasimova commented on .wiki/repositories/conan.md:
````diff
@@ -0,0 +1,22 @@
+### Client configuration
+
+```bash
+conan remote add conan-test http://artipie.artipie:9300 False
````
I'd suggest to add more detailed description: what is `conan-test`? What `False` mean?

olenagerasimova reviewed (commented):
@ChGen thanks, just one cosmetic suggestion

ChGen commented:
@olenagerasimova , please review, I extended wiki.

olenagerasimova reviewed (approved):
@ChGen thanks, approving
